AzureData/Source/AzureData.swift

```diff
@@ -79,9 +79,13 @@ public func get (databaseWithId databaseId: String, callback: @escaping (Respons
 }
 
 /// Delete a Database
-//public func delete (_ database: Database, callback: @escaping (DataResponse) -> ()) {
-//    return DocumentClient.default.delete (database, callback: callback)
-//}
+public func delete (_ database: Database, callback: @escaping (DataResponse) -> ()) {
```

Comment:
I don't believe this override is needed as it's already handled by the generic func here:
https://github.com/Azure/Azure.iOS/blob/31d13a1306d11fe35fb23730931318c67fe0d3f5/AzureData/Source/AzureData.swift#L484-L486

Reply:
Oh! That's right. Thanks for catching that.